Source/Utilities/InitInstance.lua

```diff
@@ -0,0 +1,5 @@
+function Auctionator.Utilities.InitInstance(mixin, ...)
```

Comment:
This function already exists in the Blizz API (see Mixin.lua@CreateAndInitFromMixin)